lighthouse-core/audits/non-composited-animations.js

```diff
@@ -27,6 +27,16 @@ const UIStrings = {
     =1 {Unsupported CSS Property: {properties}}
     other {Unsupported CSS Properties: {properties}}
   }`,
+  /** Name of a compositor failure reason where the transform being animated depends on the box size. */
+  transformDependsBoxSize: 'Transform related property depends on box size',
+  /** Name of a compositor failure reason where the filter being animated may move pixels. */
+  filterMayMovePixels: 'Filter related property may move pixels',
```

Comment:
```suggestion
  filterMayMovePixels: 'Filter-related property may move pixels',
```

Reply:
I don't really know what this means though

Reply:
If the filter is blur, box-reflect or drop shadow, the animation is not compositable.